.github/workflows/android_apk_build_dev.yaml

```diff
@@ -29,15 +28,16 @@ jobs:
         if: job.status == 'success'                                                           # wait until environment is ready
         run: |
           cd android
-          wget https://eaterslab.herokuapp.com/api.yaml                                       # get current api specification from server
+          wget https://eaterslab-dev.herokuapp.com/api.yaml                                       # get current api specification from server
           chmod +x ./api_generate_kotlin.sh
           ./api_generate_kotlin.sh api.yaml
           export GRADLE_USER_HOME=$(pwd)/.gradle
           chmod +x ./gradlew
           ./gradlew assembleDebug                                                             # run gradle compilation process in android project directory
-          curl -H 'Authorization: Token ${{ secrets.TOKEN_ADMIN_DEV }}' -X PUT --data-binary @./app/build/outputs/apk/debug/app-debug.apk https://eaterslab-dev.herokuapp.com/api/beta/upload/artifacts/beta/file/ # send file to server
+          curl -H 'Authorization: Token ${{ secrets.TOKEN_ADMIN_DEV }}' -X PUT --data-binary @./app/build/outputs/apk/debug/app-debug.apk https://eaterslab-dev.herokuapp.com/api/beta/upload/artifacts/beta/file/ # send file to server dev
+          curl -H 'Authorization: Token ${{ secrets.TOKEN_ADMIN }}' -X PUT --data-binary @./app/build/outputs/apk/debug/app-debug.apk https://eaterslab.herokuapp.com/api/beta/upload/artifacts/beta/file/ # send file to server 
```

Comment:
API is fetched from dev and then APK goes to main (release?) server. Is this expected? I would expect this APK is not interoperable with different API versions.

Reply:
If app works with any two versions, can we add a comment here?

Reply:
I wanted to have two version of apps on every server but thanks to you I realized it's not a good way to go. Remove debug apk from release server and release apk from dev server